README.md

```diff
@@ -9,6 +9,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [React India](https://www.reactindia.io/) | 26-28 Sep | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more. | No |
+| [ODSC INDIA • 2019](https://india.odsc.com/) | 7-10 Aug | Bangalore | The Largest Applied Data Science & AI Conference returns to India! | No |
```

Comment:
This repo only accepts conferences with Code of Conducts and I can't find a CoC at the conference website. If you could point me to a link for the CoC, that'd be great.